client/js/libs/handlebars/friendlydate.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+
+const moment = require("moment");
+
+module.exports = function(time) {
+	// See http://momentjs.com/docs/#/displaying/calendar-time/
+	return moment(new Date(time)).calendar(null, {
+		sameDay: "[Today]",
+		lastDay: "[Yesterday]",
+		lastWeek: "L", // Locale
```

Comment:
I kind of find using locale aware date formatting a bit shit. What if we formatted like `April 22, 2017`?